google/cloud/universe_domain/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# ~~~
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+include(GoogleCloudCppLibrary)
+
+if (BUILD_UNIVERSE_DOMAIN_DEMOS AND GOOGLE_CLOUD_CPP_ENABLE_CXX_EXCEPTIONS)
```

Comment:
just so I am clear, we never set `BUILD_UNIVERSE_DOMAIN_DEMOS` in our CI, right?